service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/Job.java

```diff
@@ -22,59 +23,27 @@
     private String jobId;
     private JobClient jobClient;
 
-    /**
-     * CONSTRUCTOR
-     *
-     * @param connectionString is the IoTHub connection string.
-     * @throws IOException This exception is thrown if the object creation failed
-     */
-    Job(String connectionString) throws IOException
```

Comment:
This class is not the actual JobClient, for reference. There is a class called "JobClient" that is the more interesting one.

Reply:
This class is mostly used as a return type from existing APIs like "twinClient.scheduleTwinJob(...)"